<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>ã„´
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>940</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Camera Analysis App</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <!-- Home Page -->
      <widget class="QWidget" name="homePage">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QLabel" name="teamLogo">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>400</height>
           </size>
          </property>
          <property name="text">
           <string>Team Logo</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="btnToCamera">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Start Camera</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <!-- Camera Page -->
      <widget class="QWidget" name="cameraPage">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QFrame" name="cameraFrame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="rightPanelLayout">
          <item>
           <widget class="QListWidget" name="frameList">
            <property name="minimumWidth">
             <number>200</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnComplete">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>Complete</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <!-- Analysis Page -->
      <widget class="QWidget" name="analysisPage">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="topLayout">
          <item>
           <widget class="QLineEdit" name="usernameInput">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>Username</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnToHome">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Home</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="textBox"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="chartsLayout">
          <item>
           <widget class="QFrame" name="chart1">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>200</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="chart2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>200</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="chart3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>200</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
</ui>